CONTRIBUTING.md

```diff
@@ -1,11 +1,39 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated!
+Every little bit helps, and credit will always be given.
+
 ## Contributing Guidelines
 
-- If you'd like to make a pull request to contribute, please keep it simple. If you use some higher level techniques I don't understand, I'm not going to approve it because I won't be able to maintain it.
+- If you'd like to make a pull request to contribute, please ***keep it simple***. If you use some higher level techniques I don't understand, I'm not going to approve it because I won't be able to maintain it.
 
-- Please don't make a pull request with a bunch of changes in syntax just for the sake of 'best practices' (ex: changing "if blah == False" to "if not blah"). Unless something makes a difference to performance frankly I don't care.
+- *Please* ensure all code has been formatted by [Black](https://pypi.org/project/black/) ([Usage Guide](https://black.readthedocs.io/en/stable/getting_started.html#basic-usage)). This helps to maintain code readability and keep a consistent style across all files. This is optional for you to do.
```

Comment:
It is not fine as is, my reasoning stated in the PR description:
```Remove unnecessary line from CONTRIBUTING.md -- "Please don't make a pull request with a bunch of changes in syntax just for the sake of 'best practices' [...] Unless something makes a difference to performance frankly I don't care." -- There are plenty of PRs that get accepted that do not make the slightest difference to performance, and this line pushes away many from contributing. ( https://github.com/ThioJoe/YT-Spammer-Purge/discussions/194 : "So I guess I won't bother. Too bad." https://github.com/ThioJoe/YT-Spammer-Purge/pull/489 : "I personally don't think this project at this state is salvageable, especially given ThioJoe's attitude." )```

Reply:
Well, ok..

Reply:
If a PR is a tiny change with no difference, it will be denied. Ex. Change `is` to `==`
This should be common sense; The line should therefore be redundant, aside from making the project seem hostile to small change.
If one cannot make a big change, and one cannot make a small change, what is one to do?